File: src/main/javascript/fizzbuzz/fizzbuzz.js
var fizzbuzz;
for (var i = 1; i < 101; i++) {
    fizzbuzz = "";
    fizzbuzz += (i % 3 == 0) ? "Fizz" : "";
    fizzbuzz += (i % 5 == 0) ? "Buzz" : "";
    console.log((fizzbuzz == "") ? i : fizzbuzz);
}

File: src/test/java/FizzBuzz.java
public class FizzBuzz {
	public static void main(String[] args) {
		for (int i = 1; i < 101; i++) {
			String s = "";
			s += i % 3 == 0 ? "Fizz" : "";
			s += i % 5 == 0 ? "Buzz" : "";
			System.out.println(s.isBlank() ? i : s);
		}
	}
}
